src/components/RegistrationForm/TitleHaveAccount/TitleHaveAccount.styled.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export const StyledText = styled.p`
+  //margin-top: 17px;
+
```

Comment:
Maybe also remove and delete an empty line

Reply:
fixed